CeedlingSettings.py

```diff
@@ -84,56 +106,63 @@ def update_cache(self, window):
                 raise IOError("Configuration file 'project.yml' not found.")
 
         # Update if cache is out of date or doesn't exist
-        last_mod = os.stat(project_file).st_mtime
-        cached_mod = self._cache_get("last_modified")
+        file_mod_time = os.stat(project_file).st_mtime
+        cached_mod_time = self._cache_get("last_modified")
 
-        if (cached_mod is None) or (last_mod > cached_mod):
-            self._cache_set(self.project_file_parse(project_file))
+        if (cached_mod_time is None) or (file_mod_time > cached_mod_time):
+            self._cache_set(self._project_file_parse(project_file))
             self._cache_set(
```

Comment:
Optional: When taking inspiration from the default `dict` method, this could be called `_cache_update`.

Reply:
Makes sense. I had `_cache_get`, `_cache_set`. `_update_cache` was consistent. 
Fixed some other naming issues in the same file.